lib/record-serialization-fallback.js

```diff
@@ -261,8 +258,10 @@ JsrsParser.prototype.skipComments = function() {
 // Skip the body of a single-line comment (i.e. the part after //)
 //
 JsrsParser.prototype.skipLineCommentBody = function() {
-  while (!this.isNewline(this.lookahead()) &&
-          this.lookahead() !== undefined) {
+  while (
```

Comment:
Ain't it the same as  [lib/cli/utils.js#90](https://github.com/metarhia/jstp/blob/86218918a4d906649d8c173239b80f350692d8cf/lib/cli/utils.js#L90) and [comment](https://github.com/metarhia/jstp/pull/244/files#r126899858)?